Refactor: "룸 출력을 리스트로 변경"

hana2set opened the pull request:
무한스크롤 방식 채택을 위해 리스트 방식으로 변경
findAllSearchFilter 성능개선
호환성 문제로 jpaQueryFactory 변경 (transform 함수 사용 불가 현상)
**주의 - 프런트와 같이 변동사항 있음**

Resolves: #187
Resolves: #153
Resolves: #56

hana2set committed:
Refactor: "룸 출력을 리스트로 변경"

무한스크롤 방식 채택을 위해 리스트 방식으로 변경
findAllSearchFilter 성능개선
호환성 문제로 jpaQueryFactory 변경 (transform 함수 사용 불가 현상)

Resolves: #187
Resolves: #153
Resolves: #56

Kim-Jong-Gyu commented on src/main/java/com/example/airdns/domain/room/dto/RoomsSearchConditionDto.java:
pazeSize를 pageSize 로 바꾸시면 될거 같습니다!!

hana2set committed:
Refactor: "룸 출력을 리스트로 변경"

무한스크롤 방식 채택을 위해 리스트 방식으로 변경
findAllSearchFilter 성능개선
호환성 문제로 jpaQueryFactory 변경 (transform 함수 사용 불가 현상)

Resolves: #187
Resolves: #153
Resolves: #56

parkjihwanKr reviewed (approved):
고생하셨습니다!
